structured-query/documentation/2021_10_18StructeredQueryV2Documentation.md

```diff
@@ -85,26 +82,45 @@ A valueFilter of type quantity-comperator can be applied to all numeric criterio
 
 A valueFilter of type quantity-range can be applied to all numeric criterion concepts to validate if a value is within the boundaries of the defined min and max values. Again a unit can be given.
 
-## ConceptValueFilter
+### DateTime-comperator
+
+A valueFilter of type datetype can be applied to all datetime criterion concepts. It allows to use the 
```

Comment:
IHMO the type should be called `date-time-comparator` and not `datetype` to be on par with the `quantity-comparator` type name.